src/Image.cc

```diff
@@ -268,6 +268,12 @@ NAN_SETTER(Image::SetOnload) {
   if (value->IsFunction()) {
     Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
     img->onload = new Nan::Callback(value.As<Function>());
+  } else if (value->IsNull()) {
+    Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
+    if (img->onload) {
+      delete img->onload;
+    }
+    img->onload = NULL;
```

Comment:
Sorry, one last thing again :laughing: 

Could you replace `NULL` with `nullptr` since we are always building with C++11. This is better because `NULL` is just a macro for `0` whereas `nullptr` is a value of a the special type `std::nullptr_t`. Because of that it can never accidentally be used as an int, or really anything other than a pointer.

I promise I will merge and release after this :dancer: